apps/api/src/item/database/item-database.service.ts

```
@@ -0,0 +1,39 @@
+
+  async addVerifyItem(data: { itemName: string; originalString: string; owner: string }) {
+    if (!data.itemName || !data.originalString || !data.owner) {
+      throw new Error('Missing required fields.');
+    }
+    // If owner is a name, fetch UUID from Mojang API
+    if (typeof data.owner === 'string') {
+      try {
+        const resp = await fetch(`https://api.mojang.com/users/profiles/minecraft/${data.owner}`);
+        if (resp.ok) {
+          const mojang = await resp.json();
+          if (mojang && mojang.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)